README.md

```diff
@@ -634,8 +634,46 @@ docker run --volumes-from dcert -e DOCKER_HOST=$DOCKER_HOST -e DOCKER_TLS_VERIFY
 Different CI/CD services have different containerized environment designs that impose various restrictions that may impact the ability of the main app to communicate with the sensor app embedded in the temporary container Slim creates. Try adjusting the values for the `--sensor-ipc-mode` and `--sensor-ipc-endpoint` flags. This [`Google Cloud Build`](https://medium.com/google-cloud/integrating-dockerslim-container-minify-step-on-cloud-build-64da29fd58d1) blog post by Márton Kodok is a good reference for both of those flags. 
 
 ### Using `*-file` Flags
+- There are several flags that accept file paths (`--include-path-file`, `--compose-file`, `--http-probe-cmd-file`, etc). You need volume mount the location of the referenced paths or the file paths themselves when you use the containerized version of Slim because the Slim app container won't have accept to the referenced files otherwise.
 
-There are several flags that accept file paths (`--include-path-file`, `--compose-file`, `--http-probe-cmd-file`, etc). You need volume mount the location of the referenced paths or the file paths themselves when you use the containerized version of Slim because the Slim app container won't have accept to the referenced files otherwise.
+### CI/CD INTEGRATIONS
+
+#### Integrating Slim in Jenkins
```

Comment:
Need more context in these instructions... Where's the information about Jenkinsfile and how it needs to be customized? It's not clear where the provided snippets are supposed to go.

I expected to see a Jenkinsfile snippet like this (with the info about where to put it and how to customize it):

```
        stage("Build and Slim Docker Image") {
            steps {
                script {
                    echo "building and slimming docker image..."
                    sh 'docker build -t node_alpine:latest'
                    sh 'slim build --target node_alpine:latest --tag node_alpine:slim-$BUILD_NUMBER'
                    }
                }
            }
        }
```

Reply:
Hello @kcq, thanks for the feedback, I have rewritten the entire CI/CD integrations section in the repo README, kindly review.